packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -0,0 +1,136 @@
+import { ICheckStyleProps, ICheckStyles } from './Check.types';
+import {
+  concatStyleSets,
+  getTheme,
+  HighContrastSelector,
+  IStyle,
+  ITheme,
+} from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+
+const DEFAULT_CHECKBOX_HEIGHT: string = '18px';
+
+export const getStyles = (
+  props: ICheckStyleProps
+): ICheckStyles => {
+  const {
+    checkBoxHeight = DEFAULT_CHECKBOX_HEIGHT,
+    theme,
+    checked
+  } = props;
+
+  const { palette, semanticColors } = theme;
+
+  const _sharedCircleCheck: IStyle = {
```

Comment:
nit: We typically only prepend the _ when it's a private member of a class but not a local variable.

Reply:
Commit pushed